### PR TITLE
#5038 Redesign and display the full error in Property view's Nodes panel

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5038)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/MetricsPanel.js [hunk 1]
```diff
@@ -51,32 +51,34 @@ const MetricsPanel = props => {
...
-                        </Typography>
-                      }
-                    />
-                  );
-                })}
-              </React.Fragment>
-            );
-          })}
+          {Object.keys(metrics)
+            .sort()
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/MetricsPanel.js:55]: Metrics are not sorted...

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodeErrorDialog.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <>
+      <Dialog
+        onClose={onClose}
+        maxWidth="md"
+        open={isOpen}
+        showActions={false}
+        title={error.title}
+      >
+        <ReactJson
+          src={error.message}
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodeErrorDialog.js:38]: Display the error just like event log component. But the error here is a string, so the output is much simpler

**Comment by konekoya** [comment]
Testing this with our Demo env

Imgur

**Comment by eechih** [comment]
螢幕快照 2020-05-26 下午3 23 38

稍微調整一下

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js [hunk 3]
```diff
@@ -54,12 +54,12 @@ const NodePanel = props => {
         </ExpansionPanelSummary>
         <ExpansionPanelDetails>
           <NodeList
-            heading="Masters"
+            heading={`Masters (${masterNodes.length})`}
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: cc @eechih :)

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: Good.

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: 請問 ohara-demo-02 上方的 `Name` 有拿掉嗎？

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: Oh, oops. Didn't notice that, sorry XD

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: No worries

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/NodePanel.js:57]: Updated. Sorry for my sloppiness 😅

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**